app/models/hcb_code.rb

```diff
@@ -133,6 +133,11 @@ def amount_cents
     end
   end
 
+  def display_amount
+    return disbursement.amount.abs if disbursement?
+    return amount.abs
+  end
```

Comment:
I believe we should be using `amount_cents_by_event` instead of creating a new event